src/test/java/io/bittiger/ads/activity/AdsDaoTest.java

```diff
@@ -13,11 +13,14 @@ public void testAdsDao() {
         long expectedAdId = 1l;
         long expectedCampaignId = 2l;
         long expectedBid = 3l;
+        String[] keywords = {"key1"};
 
         Ad ad = new Ad();
         ad.setAdId(expectedAdId);
         ad.setCampaignId(expectedCampaignId);
         ad.setBid(expectedBid);
+        ad.setKeywords(keywords);
+
```

Comment:
You'd better add a assert below to test whether the Keywords are the same as your input.
And don't forget to delete this extra line break. 😉